e2e/rook-test-framework/README.md

```diff
@@ -0,0 +1 @@
+# rook-test-framework
```

Comment:
for the folder structure, i would recommend a few changes such as removing the `rook` prefix. It's redundant with the rook repo. Names can be concise. For example, `e2e` already implies tests, right? What if you renamed these?
  - `rook-test-framework` to `framework`
  - `rook-infra-manager` to `manager`
  - `test-data` to `data`